velox/expression/tests/MapViewTest.cpp

```diff
@@ -136,32 +136,6 @@ class MapViewTest : public functions::test::FunctionBaseTest {
     }
   }
 
-  // MapView can be seen as std::vector<pair<key, value>>.
-  void indexedLoopTest() {
-    auto mapVector = createTestMapVector();
-    DecodedVector decoded;
-    exec::VectorReader<Map<int64_t, int64_t>> reader(
-        decode(decoded, *mapVector.get()));
-
-    for (auto i = 0; i < mapsData.size(); ++i) {
-      auto mapView = read(reader, i);
-      auto it = mapsData[i].begin();
-      int count = 0;
-      ASSERT_EQ(mapsData[i].size(), mapView.size());
-      for (int j = 0; j < mapView.size(); j++) {
-        ASSERT_EQ(mapView[j].first, it->first);
```

Comment:
Why not keep this test and just change it to use atIndex instead of [] now that you've changed the meaning.

I may have missed it and this test is duplicating something, but it still seems useful for testing the atIndex logic.